test/sequential/test-http-pipeline-flood.js

```diff
@@ -8,38 +18,49 @@ switch (process.argv[2]) {
   case 'child':
     return child();
   default:
-    throw new Error('wtf');
+    throw Error(`Unexpected value: ${process.argv[2]}`);
```

Comment:
I vaguely remember a PR to make all Errors be called with new. May be worthwhile to make it consistent?